api/apps/api/src/modules/scenarios/scenarios.controller.ts

```diff
@@ -170,4 +177,53 @@ export class ScenariosController {
       await this.service.getFeatures(id),
     );
   }
+
+  @ApiOkResponse({
+    type: ScenarioSolutionResultDto,
+  })
+  @ApiImplicitQuery({
+    name: 'best',
+    required: false,
+    type: Boolean,
+  })
+  @ApiImplicitQuery({
+    name: 'most-different',
+    required: false,
+    type: Boolean,
+  })
```

Comment:
`ApiImplicitQuery` is deprecated/not in API anymore, you should use `ApiQuery`

Reply:
Thanks for catching it, fixed in https://github.com/Vizzuality/marxan-cloud/pull/264/commits/78de3b1034837a70b3563efe89283787a28f90ad